resources/frameworks.yaml

```diff
@@ -195,6 +203,17 @@ TPOT:
 ### TimeSeries AutoML frameworks ###
 ####################################
 
+AutoPyTorchTS:
```

Comment:
@sebhrusen should we work with a separate framework file for timeseries frameworks/baselines (even if they might reference the same `frameworks/` implementation with a different setting)? I think I would prefer that.

Reply:
yes, makes sense until we have plans to officially include TS in the benchmark paper